docs/pages/setup/reference/networking.mdx

```diff
@@ -54,7 +54,16 @@ Environment="NO_PROXY=localhost,127.0.0.1,192.168.0.0/16,172.16.0.0/12,10.0.0.0/
 
 ## Ports
 
-Teleport services listen on several ports. This table shows the default port numbers.
+Teleport services listen on several ports. This table shows the default port numbers.  For Teleport Cloud use the below recommendation to get the ports
+that are assigned for your proxy.
+
+<Admonition
+  type="tip"
+  title="Note"
+>
+  To get a listing of the assigned ports for a given proxy use `curl https://teleport.example.com:443 | jq`.  Note
```

Comment:
Do you mean to hit `/webapi/ping`?

If you hit the root like this you'll just get HTML.

Reply:
@zmb3 thank you yes, that was intended. corrected.